README.md

```diff
@@ -215,6 +216,56 @@ def about(conn, params) do
 end
 ```
 
+## Dynamic Pages
+
+Thesis supports users creating and deleting dynamically routed pages. These
+differ from static pages in that they are routed by Thesis rather than Phoenix,
+and live only in your database. They can be rendered with different templates.
+
+![add new page screenshot](https://cloud.githubusercontent.com/assets/1479215/17272456/9ac06bb4-564b-11e6-8990-c7964f5ebd63.png)
+
+To enable dynamic pages, add (or uncomment) this in your `config/config.exs` file:
+
+```elixir
+config :thesis, :dynamic_pages,
+  view: <MyApp>.DynamicView,
+  templates: ["index.html", "otherview.html"],
+  not_found_view: <MyApp>.ErrorView,
+  not_found_template: "404.html"
+```
+
+Replace `<MyApp>` with your app name. Use any view you want, and put any templates
+contained in that view that you want to make available in the `templates` list.
+These will be displayed as a drop-down to the user when they are creating the new
+dynamic page.
+
+You'll also need to make one change to your router.ex and a controller of your
+choice.
+
+```elixir
+# web/router.ex
+
+  get "/*path", <MyApp>.PageController, :dynamic
+
+# web/controllers/page_controller.ex (or similar)
+
+  def dynamic(conn, _params) do
+    render_dynamic(conn)
+  end
+```
+
+You can pass in a default template (otherwise, it'll use the first template
+option in your config) with `render_dynamic(conn, template: "index.html")`.
+
+You can choose to make only a portion of your website support static pages by
+routing more specifically. For example, if you want a blog section:
+
+```elixir
+# web/router.ex
+
+  get "/blog/*path", <MyApp>.BlogController, :dynamic
+```
```

Comment:
I haven't actually tested this, and it won't prevent you from making dynamic pages elsewhere. It just won't render them.